platform/kobo/koreader.sh

```diff
@@ -80,6 +80,9 @@ if [ ! -n "${PLATFORM}" ] ; then
 fi
 # end of value check of PLATFORM
 
+# Remount SD to RW, ignore errors since we may not have sd card
```

Comment:
If [ -f won't work, the /mnt/sd folder always exists, even sd card is not mounted. And one more concern is, even /dev/mmcblk1p1 (sd card block device) is existing, we may still not able to mount it as RW. i.e. the file system may not be supported. But if the file system is supported, remount as RW should always succeed. So simply ignoring the error may be the most simply way to solve most cases.